### PR TITLE
Styles file

### LINKED ISSUE #31: Define styles globally
Create a file `styles.dart` in `lib` folder containing all the styles we would use in the app in different places. This way we won't have to define styles in all the different places in the UI.
For instance, this file could contain all the different colours we'd use in the app like
- background colour
- primary colours
- secondary colours

This file would also include all the different styles for consistent reusable components like 
- text style
  - styles for the app title
  - styles for headings
  - styles for body
  - other text styles which are used consistently in the app 
- shadows
- elevations
- icon styles

This list is just for reference and not exhaustive.

### PR DESCRIPTION
**Pull request by Monik09**
Fixes #31 
Added global variables for text colors, background color, icon style, relative image address,shadows and text style according to new design model.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"java.configuration.updateBuildConfiguration": "interactive"
+}
```
Comment (thecodepapaya) [inline .vscode/settings.json:3]: Are you sure you wanted to add these files to version control?

File: android/.settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.project.dir=
+eclipse.preferences.version=1
+gradle.user.home=
+java.home=C\:/Program Files/Java/jdk-13.0.2
+jvm.arguments=
+offline.mode=false
+override.workspace.settings=true
+show.console.view=true
+show.executions.view=true
```
Comment (thecodepapaya) [inline android/.settings/org.eclipse.buildship.core.prefs:13]: you might want to recheck one of your `.gitignore` files

**Review by thecodepapaya — CHANGES_REQUESTED** [review]
Please recheck your .gitignore files, there are some files which are not required to be added to the version control

File: .vscode/settings.json (hunk 1 above)
Comment (Monik09) [reply .vscode/settings.json:3]: No, they are useless might have been installed by android studio.

**Review by thecodepapaya — APPROVED** [review]

**Pull request merged**